app/src/main/res/layout/activity_scoreboard.xml

```diff
@@ -4,31 +4,48 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".ScoreboardActivity">
+    tools:context=".scoreboard.ScoreboardActivity">
 
     <ScrollView
-        android:id="@+id/scroller"
+        android:id="@+id/scrollerOwnRank"
         android:layout_width="match_parent"
         android:layout_height="0dp"
-        app:layout_constraintBottom_toTopOf="@+id/leaveScoreboardButton"
+
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.5"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent">
 
-        <TableLayout
-            android:id="@+id/scoreTable"
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/recyclerViewOwnRank"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content" />
+    </ScrollView>
+
+    <ScrollView
+        android:id="@+id/scroller"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/scrollerOwnRank">
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/recyclerViewScoreboard"
             android:layout_width="match_parent"
             android:layout_height="wrap_content" />
     </ScrollView>
 
-    <Button
-        android:id="@+id/leaveScoreboardButton"
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/floatingActionButtonReturn"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/leave_scoreboard"
+        android:layout_marginEnd="8dp"
+        android:layout_marginBottom="32dp"
+        android:clickable="true"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.5"
-        app:layout_constraintStart_toStartOf="parent" />
+        app:srcCompat="@drawable/ic_baseline_arrow_back_24"
```

Comment:
We will have one day to discuss with the team to decide on some guidelines for the typical margins/colors to use in the app so we use all the same. We will do that later apparently thought.